apps/site-landing/lang/en.json

```diff
@@ -341,10 +341,6 @@
     "defaultMessage": "Success: check your emails",
     "description": "site.subscribe.success: Message we give a user when the subscription is a success"
   },
-  "tk3UgV": {
```

Comment:
text should not be transalted

Reply:
Transalted: for a second I thought I was learning a new word today